specs/svg-native/index.bs

```diff
@@ -128,128 +190,88 @@ Only the following units are supported:
 - ''cm''
 - ''in''
 
+The 'transform' property is only supported by SVG Native on the following elements:
```

Comment:
What is the purpose of this exception? What are you excluding & why?

Reply:
From what I could tell, SVG 2 allows the transform attribute on any element. However, SVG 1.1 doesn't allow that. Allowing it anywhere makes implementation more difficult and confusing; this restriction simplifies it.